app/src/main/java/com/example/househomey/filter/ui/MakeFilterFragment.java

```diff
@@ -31,4 +31,12 @@ public void getFilterInput() {
         filterCallback.onFilterApplied(makeFilter);
         dismiss();
     }
+
+    /**
+     * Deletes the currently applied "Make" filter on the home fragment.
+     */
+    @Override
+    public void resetFilter() {
+        filterCallback.onFilterReset(MakeFilter.class);
```

Comment:
```filterCallback.onFilterReset(this);```

Reply:
`this` here is a `MakeFilterFragment` and not a `MakeFilter`